plugins/woocommerce-blocks/assets/js/atomic/blocks/product-elements/image/block.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "woocommerce/product-image",
+	"version": "1.0.0",
+	"title": "Product Image",
+	"description": "Display the main product image.",
+	"category": "woocommerce-product-elements",
+	"attributes": {
+		"showProductLink": { "type": "boolean", "default": true },
+		"showSaleBadge": { "type": "boolean", "default": true },
+		"saleBadgeAlign": { "type": "string", "default": "right" },
+		"imageSizing": { "type": "string", "default": "single" },
+		"productId": { "type": "number", "default": 0 },
+		"isDescendentOfQueryLoop": { "type": "boolean", "default": false },
+		"isDescendentOfSingleProductBlock": {
+			"type": "boolean",
+			"default": false
+		},
+		"width": { "type": "string" },
+		"height": { "type": "string" },
+		"scale": { "type": "string", "default": "cover" },
+		"aspectRatio": { "type": "string" }
```

Comment:
Nit pick: The format of this is different from other `block.json` files in the repo.

Reply:
> Nit pick: The format of this is different from other block.json files in the repo.

Agreed. But also, the new format was only introduced recently with [v2 -> v3 update](https://github.com/woocommerce/woocommerce/pull/48720/files).

As you mentioned in [comment](https://github.com/woocommerce/woocommerce/pull/48720#discussion_r1716277597):

> We don't have a configuration for JSON, which I think we should add one, using prettier.

I agree that's the action we should take to unify `block.json` files.